modules/nw-dns-cache-tuning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-cache-tuning_{context}"]
+= Tuning the CoreDNS cache
+
+You can configure the maximum duration of both sucessful (positive) or unsuccessful (negative) caching done by CoreDNS. Tuning the duration of caching of DNS query responses can reduce the load for any upstream DNS resolvers.
+
+.Procedure
+
+. Edit the DNS Operator object named `default` by running the following command:
++
+[source,terminal]
+----
+$ oc edit dns.operator.openshift.io/default
+----
+
+. Modify the time-to-live (TTL) caching values:
++
+.Configuring DNS caching
+[source,yaml]
+----
+spec:
+  cache:
+    successTTL: 1h <1>
+    denialTTL: 0.5h10m <2>
+----
++
+<1> The string value `1h` is converted to its respective number of seconds by CoreDNS. If this field is left empty, the value is assumed to be `0s` and the cluster will use the internal default value of `900s` as a fallback.
```

Comment:
For the internal default values, we'll have to keep an eye on these in the future because they're subject to change so the docs will need to be updated accordingly if/when that happens.